symplyphysics/symbols/chemistry.py

```diff
@@ -20,3 +20,26 @@
 **Amount of substance** in a given sample of matter is defined as a ratio between the number of elementary
 entities and the Avogadro constant.
 """
+
+density_of_states = SymbolNew("D", 1 / units.volume)
+r"""
+The **density of states** of a system describes describes the number of allowed modes or states
+per unit energy range.
+"""
+
+band_gap = SymbolNew("E_g", units.energy, display_latex="E_\\text{g}")
+"""
+A **band gap**, or **energy gap**, is an energy range in a solid where no electronic states exist. 
+"""
+
+work_function = SymbolNew("W", units.energy)
+"""
+**Work function** is the minimum thermodynamic work (i.e., energy) needed to remove an electron from
+a solid to a point in the vacuum immediately outside the solid surface.
+"""
+
+drift_velocity = SymbolNew("u", units.velocity)
```

Comment:
Maybe use normal velocity?
Or add `average_velocity` instead, for more general usage?

Reply:
It's not just any average velocity, it's used specifically for charged particles in the electric field. IMO if people use it separately (although they understand it is an "instance" of the ordinary velocity), it might be beneficial to create a separate symbol for the ease of people's using it. Take energy, for example: it's physically unnecessary to create separate symbols for energy, work, potential energy, kinetic energy, etc, but if we were to just have ony energy symbol, the user would need to always manually clone and rename it. We simply provide a ready-to-be-used symbol for their computations.